draft-ietf-webpush-protocol.xml

```diff
@@ -1500,7 +1500,7 @@ HEADERS      [stream 82] +END_STREAM
     <section title="Acknowledgements">
       <t>
         Significant technical input to this document has been provided by Ben
-        Bangert, Peter Beverloo, Kit Cambridge, JR Conlin, Lucas Jenns, Matthew Kaufman,

```

Comment:
Frankly, I think that the situation stinks.  And we do have plans to fix it, but those plans take time.